components/observations/SimpleForm.tsx

```diff
@@ -162,6 +162,7 @@ export const SimpleForm: React.FC<{
   };
 
   const formFieldSpacing = 16;
+  const disableFormControls = mutation.isLoading || mutation.isSuccess;
```

Comment:
Is the lifetime of `mutation` bound to the lifetime of the `SimpleForm`, so the UX ends up being that you close & re-open the form to submit two observations in sequence?